best-practices/native/react-native/training/README.md

```diff
@@ -0,0 +1,147 @@
+# Table of contents
+1. [Differences between React and React Native](#main-differences-between-utilizing-react-native-vs-react)
+2. [React](#react)
+3. [React Native](#react-native)
+4. [React Components](#react-components)
+5. [React Props](#props-to-react-for-creating-props)
+6. [React State](#lets-talk-a-little-bit-about-state)
+7. [ES6](#es6)
+8. [Component Lifecycle](#speaking-of-lifecycle)
+
+## Main differences between utilizing React Native vs React
+
+React is generally described as a JavaScript library for building user interfaces by utilizing declarative components in a component-based architecture. React Native is a framework for building native applications using React. While both of these libraries use a similar approach to building user interfaces, React Native is designed to be used for native iOS and Android Development. Below, we will be discussing the main differences between React Native vs React.
+
+## React
+
+- Uses JSX elements that directly mimmick HTML elements right down to the props.
+- React is a UI layer for developers to build their applications.
+- Utilizes a virtual DOM to render components.
+- Very small learning curve if you can use ES6.
+
+## React Native
+
+- Uses JSX elements that can *include* HTML elements but generally uses specific native components.
+- React Native is a UI layer **and** a complete framework for building native applications on iOS and Android.
+- Utilizes the device API to render components.
+- Slightly more complex learning curve if you understand ES6, little-to-no learning curve coming from React.
```

Comment:
Can you add more info for developers that want a deep dive here? For example: adding links to our react and react native resources:

https://github.com/Shift3/standards-and-practices/blob/dcd03c29e48a9f0824ab01ab38fe0057a79338ac/best-practices/client-side/framework/react/README.md

https://github.com/Shift3/standards-and-practices/blob/54a24028df8c34e723f5e8399cbcd31002113232/best-practices/native/react-native/README.md